### ISSUE TITLE
Spec text inconsistency

### ISSUE THREAD

**Issue by caitp** [rwaldron/exponentiation-operator#17]
From IRC:

```
[16:18:12] <caitp> rwaldron: do you notice some inconsistency between
    http://rwaldron.github.io/exponentiation-operator/#PostfixExpression
    and http://rwaldron.github.io/exponentiation-operator/#sec-update-expressions ? 
[16:18:40] <caitp> it looks like the "diff with last spec snapshot" wasn't updated  
[16:19:07] <caitp> it's sort of confusing to point that out during review   
```

It would be easier to talk about this feature if this were corrected

**Comment by littledan** [comment]
@rwaldron @bterlson What do you think?

**Comment by caitp** [comment]
per discussion with Dan:
- V8 currently ships the ES6-specified early `ReferenceError` behaviour (so, `ReferenceError` for `--!x` even in dead code) --- presumably this doesn't break the web or there would be people upset about this.
- At some point, this proposal said prefix-count operators had an `LeftHandSideExpression` operand rather than a `UnaryExpression` operand. If this were the case, `--!x` would become a `SyntaxError` rather than a `ReferenceError`.
- This seems to be changed back to its original (ES6) state --- but there is some inconsistent text between those two URLs mentioned above, which makes it unclear.
- Either way, this is an early error, and either way, it basically matches what already happens, so it shouldn't break the web

The only issue here is fixing the editorial bug, in order to clarify whether `--!x` becomes a `SyntaxError`, or remains an early `ReferenceError`.

I am fairly confident that the proper change is fixing http://rwaldron.github.io/exponentiation-operator/#PostfixExpression to match http://rwaldron.github.io/exponentiation-operator/#sec-update-expressions

Images of what I'm talking about, because it seems to be very hard to convey this:

http://rwaldron.github.io/exponentiation-operator/#sec-update-expressions:

<img width="636" alt="screen shot 2016-02-16 at 4 57 12 pm" src="https://cloud.githubusercontent.com/assets/2294695/13092530/7908d1b8-d4ce-11e5-9cde-c62ea26b7e2a.png">

http://rwaldron.github.io/exponentiation-operator/#PostfixExpression:

<img width="672" alt="screen shot 2016-02-16 at 4 57 25 pm" src="https://cloud.githubusercontent.com/assets/2294695/13092543/86d91f28-d4ce-11e5-8400-dce451669cb6.png">

In other words, it's just a little editorial issue that makes it hard to talk about this feature during code review
